File: app/src/main/java/net/radityalabs/contactapp/presentation/factory/ToastFactory.java
package net.radityalabs.contactapp.presentation.factory;

import android.content.Context;
import android.widget.Toast;

/**
 * Created by radityagumay on 4/13/17.
 */

public class ToastFactory {

    public static void show(Context context, String message) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/net/radityalabs/contactapp/presentation/factory/ToastFactory.kt
package net.radityalabs.contactapp.presentation.factory

import android.content.Context
import android.widget.Toast

/**
 * Created by radityagumay on 4/13/17.
 */

object ToastFactory {

    fun show(context: Context, message: String) {
        Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
    }
}
